python-client/pyproject.toml

```diff
@@ -54,7 +55,7 @@ dev = [
     "pytest-xdist>=3.3.1",
     "ruff>=0.0.271",
     "mlflow>2",
-    "black[d]>=22.12.0",
```

Comment:
`blackd` is used by intellij, it runs as a server and then intellij will call it every time you need to run reformatting.
`black[jupyter]` doesn't ship blackd command utility